dist/PreserveModuleNamePlugin.js

```diff
@@ -73,6 +73,29 @@ function getPreservedModules(modules) {
                 m[exports.preserveModuleName] = req;
                 return true;
             }
+            // Since its relative, link it with the module alias if possible
+            const reasonRequest = r.module && r.module.rawRequest;
+            const moduleDependency = r.dependency && r.dependency.module;
+            const moduleRequest = moduleDependency && removeLoaders(moduleDependency.request);
```

Comment:
Did you consider that you're in a loop over the possibly many reasons that a module was included in the compilation?

Such reasons might not be related to Aurelia at all, like regular ES `import`. Is it correct to consider them?
What if multiple reasons lead to conflicting uses of different aliases or absence of aliases?